docs/tutorials/gpu_setup_tutorial.rst

```diff
@@ -0,0 +1,228 @@
+Part 1
+======
+
+Installing CUDA Toolkit
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The first step in utilizing the computational power of your Nvidia GPU
+is to install the CUDA toolkit. This can be done by visiting `this
+link <https://developer.nvidia.com/cuda-downloads?target_os=Windows&target_arch=x86_64&target_version=10&target_type=exe_network>`__
+and inputting your system info to download the installer. You must be
+using either Linux or Windows, and you must be using one of
+`these <https://developer.nvidia.com/cuda-gpus>`__ graphics cards. Once
+the toolkit has been installed, follow the instructions in the installer
+GUI. Once complete, restart your computer.
+
+Compatibility With PyTorch
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For MPoL purposes, we want to utilize the full extent of PyTorch
+capabilities on the GPU. To do this, open up a python-enabled terminal.
+Here we will use the Anaconda Prompt and pip to install the
+dependencies. Type in ``pip install torch torchvision`` and let the
+installation run. At this point, we now must check if everything
+installed correctly and PyTorch can see your now CUDA-enabled device.
+One can do this by running ``torch.cuda.is_available()`` as seen below:
+
+.. code:: ipython3
+
+    import torch
+    print(torch.cuda.is_available())
+
+
+.. parsed-literal::
+
+    True
+    
+
+Potential Issues
+~~~~~~~~~~~~~~~~
+
+If you complete this process and ``torch.cuda.is_available()`` returns
+False, then you may need to use a more specific installation process. Go
+to the `PyTorch Official Site <https://pytorch.org/>`__ and scroll down
+on the page until you see the **Install PyTorch** section. Input your
+specifications for your needs into this area and use the text that is
+generated for your install. In the making of this tutorial, one Windows
+10 system using a Nvidia GTX 1080 required
+``pip3 install torch==1.8.1+cu111 torchvision==0.9.1+cu111 torchaudio===0.8.1 -f https://download.pytorch.org/whl/torch_stable.html``
```

Comment:
One thing I did notice when building the docs was that this line extends past the end of the other text and bleeds onto the right side of the page. Not sure if this is an issue, or if I should just split it into two lines